website/client/src/components/groups/questSidebarSection.vue

```diff
@@ -6,11 +6,11 @@
     >
       <div class="col-12 text-center">
         <div
-          class="svg-icon"
+          class="svg-icon quest-icon color"
           v-html="icons.questIcon"
```

Comment:
Icons can be `v-once`, yeah?

Reply:
oh here yeah - will do

Reply:
I updated it but github doesn't show it o.O thats odd